### DISCUSSION TITLE
Remove tesseract package and move content to environment

### DISCUSSION THREAD

**Discussion by Levi-Armstrong** [tesseract-robotics/tesseract#470] [General]
@mpowelson  With recent changes most everything is now located in the environment class. I would like to remove this package and migrate content to tesseract_environment.

**Comment by mpowelson** [comment]
I think that is probably fine.

**Reply by Levi-Armstrong** [reply]
Now that everything is serialized into the command language the initInfo I believe is not needed. We just reset and leave the first two commands I believe. I will take a shot and see if I run into any issues.

**Reply by mpowelson** [reply]
I believe that is true.

**Comment by Levi-Armstrong** [comment]
@mpowelson I have created a PR removing the tesseract package. It was not as bad as I thought it would be. I tagged you for review.
